apps/demos/Demos/Chat/Overview/Angular/app/app.component.css

```diff
@@ -0,0 +1,20 @@
+.demo-container {
+    height: 100%;
+    padding: 30px;
```

Comment:
demo frame on the site already has the correct padding of 20px, which we need according to the Figma design. Let's remove this padding

Reply:
57acc8aa4c51d5f8d5e5517fb13187024a0be1cd